examples/CodeCoverage.cmake

```diff
@@ -0,0 +1,683 @@
+# Copyright (c) 2012 - 2017, Lars Bilke
```

Comment:
you definitely do not want this file twice (it is also in `test` directory)

Reply:
ok I take care of that